txscript/opcode.go

```diff
@@ -586,9 +586,9 @@ var opcodeArray = [256]opcode{
 	OP_UNKNOWN246: {OP_UNKNOWN246, "OP_UNKNOWN246", 1, opcodeNop},
 	OP_UNKNOWN247: {OP_UNKNOWN247, "OP_UNKNOWN247", 1, opcodeNop},
 	OP_UNKNOWN248: {OP_UNKNOWN248, "OP_UNKNOWN248", 1, opcodeNop},
+	OP_UNKNOWN249: {OP_UNKNOWN249, "OP_UNKNOWN249", 1, opcodeNop},
```

Comment:
This would introduce a hard fork.  It needs to remain `opcodeInvalid`.

Reply:
Will be sure to assert modifications to upstream commits before putting up a PR. Sorry about that.